corehq/apps/users/models.py

```diff
@@ -1582,15 +1581,13 @@ def get_cases(self, deleted=False, last_submitter=False, wrap=True):
             yield CommCareCase.wrap(doc) if wrap else doc
 
     @property
-    def case_count(self):
-        result = CommCareCase.view('case/by_user',
-            startkey=[self.user_id],
-            endkey=[self.user_id, {}], group_level=0
-        ).one()
-        if result:
-            return result['value']
-        else:
-            return 0
+    def analytics_only_case_count(self):
+        """
+        Get an approximate count of cases which were last submitted to by this user.
+
+        This number is not guaranteed to be 100% accurate since it depends on a secondary index (sofabed)
+        """
+        return CaseData.objects.filter(user_id=self._id).count()
```

Comment:
This is likely going to be slow and since its not accurate anyway we may want to use this method of getting a count estimate from PostgreSQL: https://wiki.postgresql.org/wiki/Count_estimate

Perhaps something for later?